src/main/asciidoc/kubernetes-server.adoc

```diff
@@ -0,0 +1,106 @@
+
+The Kubernetes Data Flow Server is Spring Boot application available for http://cloud.spring.io/spring-cloud-dataflow/#platform-implementations/[download] or you can https://github.com/spring-cloud/spring-cloud-dataflow-server-kubernetes[build] it yourself.
```

Comment:
Instead of pointing to binary / build procedure? Could we not point to [getting-started](http://docs.spring.io/spring-cloud-dataflow-server-kubernetes/docs/1.7.1.RELEASE/reference/htmlsingle/#kubernetes-getting-started) directly?

Reply:
Ah, it seems like you listed it in the next paragraph. Probably better to move it up anyway.

Reply:
Flows better now I think

Reply:
done